### PR TITLE
refactor: invalid username error message to clarify the accepter format

### PR DESCRIPTION
**Pull request by PredictiveManish**
### https://github.com/dailydotdev/docs/issues/400 
This issue is solved in this PR.

### This PR changes the error message seen by user during registering and having their username.
- When people select/enter their username- It was showing username is invalid but not showing what is the requirements.
- This issue is raised in the docs repo of daily.dev.

image

### Summary-
This issue will help users to find out name format so that they can easily register and make their account freely.

### CODE REVIEW

File: packages/shared/src/graphql/common.ts [hunk 1]
```diff
@@ -119,7 +119,7 @@ export const DEFAULT_ERROR = 'An error occurred, please try again';
 
 export const errorMessage = {
   profile: {
-    invalidUsername: 'Invalid characters found in username!',
+    invalidUsername: 'Username can only contain Alphanumeric (A-z, a-z, 1234..)',
```
Comment (sshanzel) [inline packages/shared/src/graphql/common.ts:122]: There is actually a specific regex condition we use for checking usernames, and it is not limited to this.

Comment (PredictiveManish) [reply packages/shared/src/graphql/common.ts:122]: Ok, so it is automated or can be changed?

Comment (PredictiveManish) [reply packages/shared/src/graphql/common.ts:122]: image
This one but it is checking the condition and the issue was just showcasing the error message isn't these both are different? Please clear my doubt.

Comment (sshanzel) [reply packages/shared/src/graphql/common.ts:122]: The one we use for usernames is the `handleRegex`

Comment (PredictiveManish) [reply packages/shared/src/graphql/common.ts:122]: Thanks Updating and solving issue!

Comment (sshanzel) [reply packages/shared/src/graphql/common.ts:122]: The message is still inaccurate like mentioned.

Comment (PredictiveManish) [reply packages/shared/src/graphql/common.ts:122]: Sorry, due to some other work I was busy solving it in 1 hour.

Comment (PredictiveManish) [reply packages/shared/src/graphql/common.ts:122]: When i am going to testuser profile page without doing any changes I am facing this below mentioned issue
image
and in code I am facing this issue:
Screenshot 2024-10-15 122504

Comment (sshanzel) [reply packages/shared/src/graphql/common.ts:122]: All cool, no worries. Thank you for trying! 🙏

**Pull request closed without merging**